docs/connector-development/cdk-python/README.md

```diff
@@ -1,16 +1,19 @@
 # Connector Development Kit (Python)
 
+    :::info
+    Singer Taps are no longer supported and contributions of this type will not be accepted by the Airbyte team.
+    :::
+
 The Airbyte Python CDK is a framework for rapidly developing production-grade Airbyte connectors. The CDK currently offers helpers specific for creating Airbyte source connectors for:
 
 * HTTP APIs \(REST APIs, GraphQL, etc..\)
-* Singer Taps
```

Comment:
The CDK can still work with Singer. Are you saying this should be hidden/we shouldn't facilitate it for people?  Seems harmless to keep this here.

Reply:
Maybe add a note:
* Singer Taps (you can build connectors based on Singer but Airbyte doesn't accept any new singer based connectors?)